openapi/components/schemas/GatewayAccountConfig/Payretailers.yaml

```
@@ -0,0 +1,20 @@
+description: Payretailers gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: Payretailers credentials object.
+        properties:
+          shopId:
+            type: string
```

nitpick: We should add descriptions for all properties
```suggestion
            type: string
            description: ID of the PayRetailers shop.
```